packages/client/src/project/entities.ts

```diff
@@ -62,6 +62,34 @@ export function getProjectEntities (projectId: string, apiClient: AxiosInstance)
         `/projects/${projectId}/entities`, expandedEntity);
       const returnedEntity = res.data;
       return wrapEntity(returnedEntity);
+    },
+
+    // Tags
+
+    async addTags(entityId: string, tags: string[]): Promise<WrappedEntity> {
+      const _tags: string[] = Array.isArray(tags)? tags: Array.from([tags]);
```

Comment:
Isn't `Array.from([tags])` equivalent to just doing `[tags]`, since `[tags]` is already an array?